__tests__/api.test.js

```diff
@@ -132,27 +133,27 @@ describe("GET /api/articles", () => {
     });
     it("returns 200 and sorts articles by votes in ascending order with a valid topic", () => {
       return request(app)
-      .get("/api/articles?sort_by=votes&order=asc&topic=mitch")
-      .expect(200)
-      .then(({ body }) => {
-        const { articles } = body;
-        articles.forEach((article) => {
-          console.log(article.topic)
-          expect(article.topic).toBe("mitch")
-        })
-          const sortedArticles = [...articles].sort((a, b) => a.votes - b.votes);
-          expect(articles).toEqual(sortedArticles)
-          
-      })
-    })
+        .get("/api/articles?sort_by=votes&order=asc&topic=mitch")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          articles.forEach((article) => {
+            expect(article.topic).toBe("mitch");
+          });
+          const sortedArticles = [...articles].sort(
+            (a, b) => a.votes - b.votes
+          );
+          expect(articles).toEqual(sortedArticles);
+        });
+    });
     it("returns 404 for an return of 0 results for that topic", () => {
       return request(app)
-      .get("/api/articles?sort_by=votes&order=asc&topic=callum")
-      .expect(404)
-      .then(({ body }) => {
-        expect(body.msg).toBe("404: No Articles Found for Specified Topic")
-      })
-    })
+        .get("/api/articles?sort_by=votes&order=asc&topic=callum")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("404: No Articles Found for Specified Topic");
+        });
```

Comment:
This is a 404 topic does not exist. We also need another 200 with an empty array to test existent topic but no article related